source/Tests/include/TestUtilities.hh

```diff
@@ -238,51 +238,51 @@ bool checkReconstructedObject(GlobalMethodsClass const& gmc, LCCluster& thisClus
 }
 
 void initLogLevels(){
-  streamlog::out.addLevelName<marlin::DEBUG>();
-  streamlog::out.addLevelName<marlin::DEBUG0>();
-  streamlog::out.addLevelName<marlin::DEBUG1>();
-  streamlog::out.addLevelName<marlin::DEBUG2>();
-  streamlog::out.addLevelName<marlin::DEBUG3>();
-  streamlog::out.addLevelName<marlin::DEBUG4>();
-  streamlog::out.addLevelName<marlin::DEBUG5>();
-  streamlog::out.addLevelName<marlin::DEBUG6>();
-  streamlog::out.addLevelName<marlin::DEBUG7>();
-  streamlog::out.addLevelName<marlin::DEBUG8>();
-  streamlog::out.addLevelName<marlin::DEBUG9>();
-  streamlog::out.addLevelName<marlin::MESSAGE>();
-  streamlog::out.addLevelName<marlin::MESSAGE0>();
-  streamlog::out.addLevelName<marlin::MESSAGE1>();
-  streamlog::out.addLevelName<marlin::MESSAGE2>();
-  streamlog::out.addLevelName<marlin::MESSAGE3>();
-  streamlog::out.addLevelName<marlin::MESSAGE4>();
-  streamlog::out.addLevelName<marlin::MESSAGE5>();
-  streamlog::out.addLevelName<marlin::MESSAGE6>();
-  streamlog::out.addLevelName<marlin::MESSAGE7>();
-  streamlog::out.addLevelName<marlin::MESSAGE8>();
-  streamlog::out.addLevelName<marlin::MESSAGE9>();
-  streamlog::out.addLevelName<marlin::WARNING>();
-  streamlog::out.addLevelName<marlin::WARNING0>();
-  streamlog::out.addLevelName<marlin::WARNING1>();
-  streamlog::out.addLevelName<marlin::WARNING2>();
-  streamlog::out.addLevelName<marlin::WARNING3>();
-  streamlog::out.addLevelName<marlin::WARNING4>();
-  streamlog::out.addLevelName<marlin::WARNING5>();
-  streamlog::out.addLevelName<marlin::WARNING6>();
-  streamlog::out.addLevelName<marlin::WARNING7>();
-  streamlog::out.addLevelName<marlin::WARNING8>();
-  streamlog::out.addLevelName<marlin::WARNING9>();
-  streamlog::out.addLevelName<marlin::ERROR>();
-  streamlog::out.addLevelName<marlin::ERROR0>();
-  streamlog::out.addLevelName<marlin::ERROR1>();
-  streamlog::out.addLevelName<marlin::ERROR2>();
-  streamlog::out.addLevelName<marlin::ERROR3>();
-  streamlog::out.addLevelName<marlin::ERROR4>();
-  streamlog::out.addLevelName<marlin::ERROR5>();
-  streamlog::out.addLevelName<marlin::ERROR6>();
-  streamlog::out.addLevelName<marlin::ERROR7>();
-  streamlog::out.addLevelName<marlin::ERROR8>();
-  streamlog::out.addLevelName<marlin::ERROR9>();
-  streamlog::out.addLevelName<marlin::SILENT>();
+  // streamlog::out.addLevelName<marlin::DEBUG>();
```

Comment:
Is this no longer necessary? Are all the log levels no already defined?

Reply:
Indeed it is not anymore. 

See https://github.com/iLCSoft/iLCUtil/pull/14/files#diff-3bf289d9dca0cee6235c43ccae938a15R600

This is called on logger construction

Reply:
Then please get completely rid of this function instead of commenting all the lines.